gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -157,6 +168,18 @@ fn v1_routes(gateway: Gateway) -> Router {
         .layer(CorsLayer::permissive())
 }
 
+/// Creates a password hash by appending a 4 byte salt to the plaintext
```

Comment:
Nit: I know 32 bits of salt already makes rainbow tables extremely expensive, but would it be worth bumping the salt to 64 bits since it's only an extra few bytes of storage?

Reply:
I completely missed it's a number. Make that stuff a 16B array of randomness, don't think twice. :)

Reply:
Changed to 16 bytes